src/components/ChatApp/Settings/TextToSpeechSettings.react.js

```diff
@@ -210,7 +212,7 @@ class TextToSpeechSettings extends Component {
             onClick={this.resetPitch}
           />
         </div>
-        <div style={{ textAlign: 'center' }}>
+        <div style={{ textAlign: MobileView <= 520 ? 'left' : 'center' }}>
```

Comment:
MobileView return boolean, then what is this check for. Also keep camelCase standards for variable definition.